code/Justin/Python/lab_3.py

```diff
@@ -0,0 +1,37 @@
+# # Defining a function to calculate our list average
+# def avg_list():
+#     # Creating a starting total to add our list too
+#     tot = 0
+#     # Use a for loop to add each number in our list to our starting total
+#     for number in nums:
+#         number = int(number)
+#         tot += number
+#     # Printing our statement
+#     print(f" The average for the list {nums} is {round(tot/len(nums), 2)}")
+# # Creating our list
+# nums = [5, 0, 8, 3, 4, 1, 6]
+# # Calling our function
+# avg_list()
+
+# Defining a function to calculate our list average
+def avg_list():
+    # Creating a starting total to add our list too
+    tot = 0
+    # Use a for loop to add each number in our list to our starting total
+    for number in nums:
+        number = int(number)
+        tot += number
+    # Printing our statement
+    print(f" The average for the list {', '.join(nums)} is {round(tot/len(nums), 2)}")
```

Comment:
In the future you should avoid printing inside of a function, instead return the statement and print as a return of the function.

Line 25: `return f" The average for the list {', '.join(nums)} is {round(tot/len(nums), 2)}"`
Line 37: `print(avg_list)`